EventsExpress/ClientApp/src/components/unitOfMeasuring/unitOfMeasuring-edit.js

```diff
@@ -39,17 +35,31 @@ class UnitOfMeasuringEdit extends Component {
                     <div style={divStyle} className="d-flex flex justify-content-around ">
 
                         <Field
-                            className="form-control"
+                            className="form-control w-25"
                             name="unitName"
                             label="Unit name"
                             component={renderTextField}
                         />
                         <Field
-                            className="form-control"
+                            className="form-control w-25"
                             name="shortName"
                             label="Short name"
                             component={renderTextField}
                         />
+                        <Field
+                            fullWidth={true}
+                            component={renderSelectField}
+                            className=" mw-100"
+                            name="categoryOfMeasuring"
+                            label='Category'
+                        >
+                            <option aria-label="None" value="" />
+                            <option value={"Weight"}>Weight</option>
```

Comment:
The UI dictating categories is a bit bad. It means that anyone calling API can come up with a category - which is not the way when we have a static list. Also, if we store it in DB - we can release new categories much more easily.